test/contextTests.ts

```diff
@@ -63,6 +63,13 @@ function commonTests(test: ITest) {
         test.write("test");
         assert.equal(test.pen.write.callCount, 1);
     });
+
+    it("measures whitespace as 0x0", () => {
+        const m = new Measurer(test.context);
+        const { width, height } = m.measure(" ");
```

Comment:
This is not the behavior we want. The space character should have a definite width of whatever the space width is - this is important because character measurers sum up the lengths of all individual chars